src/server/index.jsx

```diff
@@ -111,7 +111,7 @@ server
     res.sendStatus(200);
   })
   .get(swRegexPath, (req, res) => {
-    const swPath = `${__dirname}/public/sw-${process.env.APP_ENV}.js`;
+    const swPath = `${__dirname}/public/sw.js`;
```

Comment:
Does this affect any of our Mozart issues / JIRA tickets / tests etc...?

Reply:
Nope, this is purely internal, Mozart will continue to route to 'sw.js' in simorgh